nengo_loihi/tests/test_neurons.py

```diff
@@ -245,7 +251,7 @@ def test_nengo_dl_neuron_grads(neuron_type, plt, allclose):
     else:
         # use the derivative of y_med (the smoothed Loihi tuning curve)
         dy_ref = np.zeros_like(j)
-        dy_ref[j > 0] = neuron_type.amplitude / (j[j > 0]*tau_ref1 + 1)**2
+        dy_ref[j > 0] = neuron_type.amplitude / (j[j > 0]*tau_ref1 + 1) ** 2
```

Comment:
PEP8 doesn't require space around all binary operators: https://www.python.org/dev/peps/pep-0008/#other-recommendations

I'm really not sure what PEP8 would recommend in this case. Because `**` is a "lower priority" operator here, you could argue that it should have these spaces. However, I think because I'm used to cases where it's higher priority (PEP8 clearly prefers `x**2 + 1` to `x ** 2 + 1`), I always do not put spaces around `**`. It looks really weird to me.

Reply:
Maybe it's partly because `**` is already two characters, so there's a bit of "space" there. With the added spaces, it just seems like so much distance, especially because in math a superscript is small and so it feels even closer to whatever it's raising to a power.

Reply:
I mean, I also don't agree with PEP8: I prefer `(a + b)*(c - d)` to `(a+b) * (c-d)`. So maybe it's correct to have the spaces here, but not in the `x**2 + 1` case. I guess I just like things to be fairly consistent, so I'm not always thinking "do I need spaces here or not"? Which is kind of the point of PEP8, but I feel like here it actually makes these decisions harder.

Reply:
Yeah, I prefer putting spaces around all operators so that I don't need to make any decisions about whether some equation will be easier to parse with certain spacing. Then since I do that, it seems more readable to me because I expect there to be spaces around all operators. I would be in favor of enforcing that with flake8, but for now the `**2` just seemed weird to me, but I can change it back, no biggie

Reply:
Yeah I prefer spaces around all operators for the same reason.  It just removes an extra decision, and makes it easy to apply automatic formatting.